.github/workflows/nerdctl.yaml

```diff
@@ -0,0 +1,109 @@
+name: Nerdctl
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'site/**'
+
+permissions:
+  contents: read
+
+jobs:
+  nerdctl:
+    name: Nerdctl
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        ipFamily: [ipv4, ipv6]
+        deployment: [singleNode, multiNode]
+        exclude:
+          - ipFamily: ipv6
```

Comment:
Can we support rootless too?

Reply:
Thanks @AkihiroSuda 

The rootless mode support has been added to the 'work items' at : https://github.com/kubernetes-sigs/kind/issues/2317#issuecomment-1788561567

I want to provide it in another PR, so that the PR will not be too large.